### DISCUSSION TITLE
How to update an already existing DID?

### DISCUSSION THREAD

**Discussion by Ajmasta** [decentralized-identity/ion-tools#22] [Q&A]
Hello,

The docs always show that you re-create a new DID before doing operations such as updating or recovering.
However, I don't understand how to fetch an already existing DID and work on it.
I am also having difficulty understanding how the SDK sends a private key or a signature to make sure that we are the owners of that DID.

Could anyone guide me on how to update an already existing DID?

Thank you

**Comment by mahoekst** [comment]
I would like to know this as well. The update sample continues after calling the create code. I would like to update a DID I created before. Not sure how to acciomplish this with these tools?
